cPP_DSC.adoc

```diff
@@ -1372,23 +1368,21 @@ _The following are the SFP-relevant security attributes that are associated with
 * _OB.Pstate - none_
 * _OB.FAACntr - none_
 * _OB.AntiReplay - none_
-* _OB.Context- none_
-+
-].
+* _OB.Context- none_].
 
-FDP_ACF.1.2:: The TSF shall enforce the following rules to determine if an operation among controlled subjects and controlled objects is allowed:
+FDP_ACF.1.2:: The TSF shall enforce the following rules to determine if an operation among controlled subjects and controlled objects is allowed: [
 
-* [_Any subject that has been authorized to perform any operation against any OB.P_SDO or OB.T_SDO object can continue to perform this operation if one of the following conditions is true:_
+* _Any subject that has been authorized to perform any operation against any OB.P_SDO or OB.T_SDO object can continue to perform this operation if one of the following conditions is true:_
 ** _The object's SDO.Reauth attribute has a value of 'none', indicating that re-authorization is not required for subsequent interactions with the SDO;_
 ** _The object's SDO.Reauth attribute has a value of 'each use', indicating that re-authorization is required for each interaction with the SDO, and the subject has supplied valid authorization data to the TOE_
 * _[assignment: rules automatically enforced by the TSF that always prohibit certain subject-object-operation actions]_
 * _[assignment: rules automatically enforced by the TSF that always permit certain subject-object-operation actions]_
 * _[assignment: rules automatically enforced by the TSF that conditionally permit certain subject-object-operation actions based on subject security attributes, object security attributes, or other conditions]_
-* _{empty}[selection: [.underline]#[assignment: any configurable rules or parameters that can be modified to affect the behavior of the Access Control SFP], no configurable rules]#_].
+* _[selection: [assignment: any configurable rules or parameters that can be modified to affect the behavior of the Access Control SFP], no configurable rules]]_].
 
 FDP_ACF.1.3:: The TSF shall explicitly authorize access of subjects to objects based on the following additional rules: [_assignment: rules, based on security attributes, that explicitly authorize access of subjects to objects_].
 
-FDP_ACF.1.4:: The TSF shall explicitly deny access of subjects to objects based on the following additional rules: [_client applications can only access their own data, [assignment: rules, based on security attributes, that explicitly deny access of subjects to objects]_].
+FDP_ACF.1.4:: The TSF shall explicitly deny access of subjects to objects based on the following additional rules: [_client applications can only access their own data, [*assignment*: rules, based on security attributes, that explicitly deny access of subjects to objects]_].
```

Comment:
This assignment should not have bolding, this is literally exactly what is in the CC:2022 SFR

Reply:
It's a bit pedantic, but not exactly. What we do here is we partially complete the assignment and replace it with another assignment (which is actually not really defined by our conventions). This is in CC:2022:

[assignment: rules, based on security attributes, that explicitly deny access of subjects to objects]

this is what we do (formatting removed):

[client applications can only access their own data, [assignment: rules, based on security attributes, that explicitly deny access of subjects to objects]].

Maybe we should rewrite this as a refinement?

The TSF shall explicitly deny access of subjects to objects*, such that client applications can only access their own data, and* based on the following additional rules: [assignment: _rules, based on security attributes, that explicitly deny access of subjects to objects_]

Reply:
The way I would read it is that we are actually making a partially filled out assignment, so the "client applications can only access their own data" should be underlined, but then leave the rest along as those are just the normal text.

So if you want to say we are replacing the assignment with itself, OK, but then since we are filling out the assignment, then that needs to be underlined according to the convention, since we are forcing something in that assignment.

So if the assignment has to be bold, then the other part needs underlining, and then I think it is fine. This is really one of those cases where we should probably have just written the SFR the way we want it instead of trying to be so generic and then having to further refine our own EXT requirement.

Reply:
This is not an EXT requirement, so we didn't write this one. In any case, I'm fine removing the bold from the *assignment* to close this out.

Reply:
The original CC:2022 SFR is:

>The TSF shall explicitly deny access of subjects to objects based on the following additional rules: [assignment: _rules, based on security attributes, that explicitly deny access of subjects to objects_].

Clearly we placed a specific thing you have to do "client applications can only access their own data" into the original assignment, which needs to be underlined, as that would be part of the "partially filled" things. 

The rest though is identical to the CC:2022 text, which to me means just leave it along. If you want to indicate it has been modified because of the partially filled part, that's OK, I can get that, but the "client applications can only access their own data" needs to be underlined since we are requiring that to be part of the assignment. 

I read that as specifying one of the rules that has to be there. You MAY have additional rules, but you have to have that one, and so that is partially filled out.

Reply:
@woodbe I agree with you, except that partially filled must be italic, not underlined, since we're partially filling an assignment.